### ISSUE TITLE
Project Collection List is not keyboard accessible

### ISSUE THREAD

**Issue by zdavis** [ManifoldScholar/manifold#2416]
The project collection list component in the sidebar should be moved over to our accessible D&D library if possible.

**Comment by dananjohnson** [comment]
Related to #2208

**Comment by zdavis** [comment]
Oh, actually a dupe of #2208 it seems
